cdk/lambdas/wdiv-s3-trigger/requirements.txt

```diff
@@ -129,9 +129,9 @@ requests-paginator==0.2.0 \
     # via
     #   -c cdk/lambdas/wdiv-s3-trigger/requirements/../../../../requirements/constraints.txt
     #   -r cdk/lambdas/wdiv-s3-trigger/requirements/base.in
-sentry-sdk==2.1.1 \
-    --hash=sha256:95d8c0bb41c8b0bc37ab202c2c4a295bb84398ee05f4cdce55051cd75b926ec1 \
-    --hash=sha256:99aeb78fb76771513bd3b2829d12613130152620768d00cd3e45ac00cb17950f
+sentry-sdk==2.23.1 \
+    --hash=sha256:2288320465065f3f056630ce55936426204f96f63f1208edb79e033ed03774db \
+    --hash=sha256:42ef3a6cc1db3d22cb2ab24163d75b23f291ad9892b1a8c44075ce809a32b191
```

Comment:
I upgraded sentry here because the version we were using didn't match the documentation, which was making things harder.